src/OpenSheetMusicDisplay/OpenSheetMusicDisplay.ts

```diff
@@ -570,6 +575,12 @@ export class OpenSheetMusicDisplay {
         if (options.autoGenerateMutipleRestMeasuresFromRestMeasures !== undefined) {
             this.rules.AutoGenerateMutipleRestMeasuresFromRestMeasures = options.autoGenerateMutipleRestMeasuresFromRestMeasures;
         }
+        if (options.cursorsOptions !== undefined)
+        {
```

Comment:
please adjust the bracket style to what we use, bracket on same line in if blocks.

Reply:
done